Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 
 name = "utp"
-version = "0.6.0"
```

Comment:
Excellent Pr, but can you split into a fix and then another for version update. The crates update should be in the form "Version change to x.x.x" then we update docs for crates.io and master properly. Sorry we need to document this better.